File: core-csharp-practice/gcr-codebase/pragramming-elements/addTwoNumbers.java

import java.util.Scanner;

public class addTwoNumbers {
    public static void main(String[] args) {
        //  Scanner for read input from the user
        Scanner sc = new Scanner(System.in);
        // take first number input
        int n = sc.nextInt();
        // take second number input
        int m = sc.nextInt();
        int sum = n+ m;
        System.out.println("Total is " + sum);
        sc.close();
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/AddTwoNumbers.cs
using System;

class AddTwoNumber
{
    static void Main()
    {
        Console.Write("Enter first number: ");
        int n = int.Parse(Console.ReadLine());

        Console.Write("Enter second number: ");
        int m = int.Parse(Console.ReadLine());

        int sum = n + m;

        Console.WriteLine($"Sum: {sum}");
    }
}
